source/OptimaJet.DWKit.StarterApplication/Services/Workflow/WorkflowProjectService.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Hangfire.Server;
+using Microsoft.EntityFrameworkCore;
+using OptimaJet.DWKit.Application;
+using OptimaJet.DWKit.Core;
+using OptimaJet.DWKit.Core.Model;
+using OptimaJet.DWKit.StarterApplication.Models;
+using OptimaJet.DWKit.StarterApplication.Repositories;
+using OptimaJet.Workflow.Core.Runtime;
+using Serilog;
+using static OptimaJet.DWKit.StarterApplication.Services.Workflow.WorkflowProductService;
+
+namespace OptimaJet.DWKit.StarterApplication.Services.Workflow
+{
+    public class WorkflowProjectService
+    {
+
+        public IJobRepository<Project> ProjectRepository { get; }
+        public WorkflowProductService WorkflowProductService { get; }
+
+        public WorkflowProjectService(
+            IJobRepository<Project> projectRepository,
+            WorkflowProductService productService
+        )
+        {
+            ProjectRepository = projectRepository;
+            WorkflowProductService = productService;
+        }
+
+        public async Task ReassignUserTasks(int projectId) {
+            var project = this.ProjectRepository.Get()
+                .Include(proj => proj.Products)
```

Comment:
I wish EF had a more ergonomic way to do this. :-\

Reply:
Are you talking about the `.Include` syntax?  Yeah, I agree.

Reply:
yeah, and having to do .Include multiple times if you want to go down sibling branches. I think I could do a static extension to wrap something that ends up doing what I've done here. but :man_shrugging:  maybe next time :D